chromium_src/components/translate/core/browser/translate_language_list.cc

```diff
@@ -13,8 +13,8 @@ namespace translate {
 
 namespace {
 const char* const kBraveDefaultLanguageList[] = {
-    "de", "en", "es", "fr", "hi", "it", "ja",      "nl",
-    "pl", "pt", "ro", "ru", "tr", "vi", "zh-Hans",
+    "de", "en", "es", "fr", "hi", "it", "ja",    "nl",
```

Comment:
a weigh formatting for clang-format.
BTW we will soon delete this file (after enabling the all languages)